bundles/azure/aso/psql/bundle/config/00-schema.yml

```diff
@@ -0,0 +1,72 @@
+#@data/values-schema
+---
+#@schema/title "ASO Controller Namespace"
+#@schema/desc "The Namespace where the Azure ASO controller is installed that should own this Azure RM resource"
+aso_controller_namespace: ""
+
+#@schema/title "Create namespace flag"
+#@schema/desc "Whether to create or not the namespace for the resources"
+#@schema/type any=False
+#@schema/default True
+create_namespace: True
+
+#@schema/title "ResourceGroup"
+#@schema/desc "Azure ResourceGroup for the servers/database resources"
+resource_group: ""
+
+#@schema/title "UseExistingResourceGroup"
+#@schema/desc "Whether to use an existing Azure resource group or not"
+#@schema/default False
+use_existing_resource_group: True
+
+#@schema/title "ResourceName"
+#@schema/desc "Name for the resources"
+#@schema/default "aso-psql"
+resource_name: ""
+
+#@schema/title "Location"
+#@schema/desc "Location where the resources will be created"
+location: ""
+
+#@schema/title "Version"
+#@schema/desc "PostgreSQL version to deploy (only 11, 12 and 13 are currently supported"
+#@schema/default "13"
+version: ""
+
+#@schema/title "AdministratorName"
+#@schema/desc "Username for the administrator user. It cannot be 'azure_superuser', 'azuresu', 'azure_pg_admin', 'sa', 'admin', 'administrator', 'root', 'guest', 'dbmanager', 'loginmanager', 'dbo', 'information_schema', 'sys', 'db_accessadmin', 'db_backupoperator', 'db_datareader', 'db_datawriter', 'db_ddladmin', 'db_denydatareader', 'db_denydatawriter', 'db_owner', 'db_securityadmin', 'public'."
+#@schema/default "myadmin"
+administrator_name: ""
+
+#@schema/title "ResourceNamespace"
+#@schema/desc "Kubernetes namespace where the Azure resources will be created"
+#@schema/default "azure-psql"
+resource_namespace: ""
```

Comment:
question: will this work if I i don't configure `resource_namespace`? I'd expect it to deploy everything into the same namespace as the `pkgi`